packages/experiments/src/components/Pagination/examples/Pagination.Basic.Example.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { Pagination } from '../Pagination';
```

Comment:
Wrong import... this code will be seen by users and it doesn't make sense why importing from a folder up...
Use instead `import { Pagination } from '@uifabric/experiments/lib/Pagination'`

Reply:
good point. Addressed.

---
In reply to: [221848420](https://github.com/OfficeDev/office-ui-fabric-react/pull/6532#discussion_r221848420) [](ancestors = 221848420)